CM-Policy.md

```diff
@@ -24,4 +24,36 @@ See the **_Applicability_** section of the [GSA IT Security Policy](http://www.g
 
 For information on roles and responsibilities, management commitment, coordination among organizational entities, compliance, reviews, and updates please see the [Technology Transformation Service's (TTS) Common Control Policy](https://github.com/18F/compliance-docs/blob/master/TTS-Common-Control-Policy.md).
 
-## Procedures
\ No newline at end of file
+## Procedures
+
+cloud.gov's specific configuration management procedures are packaged with the actual code of the cloud.gov system. Below is an overview of our procedures along with citations to controls and relevant GitHub repositories.
+
+### 
+
+The cloud.gov team maintains a [Configuration Management Plan](https://github.com/18F/cg-docs/blob/master/content/ops/configuration-management.md) that governs configuration changes. The plan outlines the procedure for to make any and all changes impacting the configuration of the system
+
+See CM-9. 
+
+Cloud Operations encodes cloud.gov's infrastructure into a set of [Terraform](https://www.terraform.io) configuration files. Terraform files are checked into [18F's cloud.gov GitHub repositories](https://github.com/18F?utf8=%E2%9C%93&query=cg), and local git repositories, in order to ensure distributed version control and availability of the code. 
+
+Cloud Operations first provisions the initial infrastructure, Concourse (a continuous integration pipelining tool), into the desired "infrastructure as a service" account, using this [procedure](https://github.com/18F/cg-provision). Concourse then ensures that all configurations of further automated deployments are controlled, whether they are the result of running Terraform or BOSH. 
```

Comment:
I feel a bit confused while reading this first sentence. Maybe "first provisions the initial infrastructure tool"? But then when I look at the procedure link, the first step says "Terraform" rather than "Concourse", so as a semi-naive reader I feel puzzled. Not sure the best way to improve this (or even if it is a high priority to improve).